test/test_helper.rb

```diff
@@ -1,11 +1,22 @@
 # frozen_string_literal: true
 
+require 'bigdecimal'
+
 original_verbosity = $VERBOSE
 $VERBOSE = nil
+
+if ENV['SKIP_SIMPLECOV'] || BigDecimal(RUBY_VERSION[0, 3]) >= BigDecimal('3.1')
+  $SKIP_SIMPLECOV = true
+end
+
 require "rubygems"
 require "pry-byebug" unless ENV["CI"]
-require "simplecov"
-require "coveralls"
+
```

Comment:
Do you have any more info on the issue you were seeing related to simplecov? I don't see immediately why the changes to Coverage would cause an issue.

Reply:
The changes makes `Coverage#start` raise an error on 2nd invocation.

See https://github.com/ruby/ruby/pull/4856/files#diff-1dc6d1837e2b8b7d8b1ae5a3067b3a677904c72dd31d0ca0b95d20f6691f7510R47

In ruby 3.0 and older, we can keep calling `Coverage#start` without raising exception.

Reply:
ahh perfect thanks, I can get a test around this and check if enabled and not call or capture the exception. Thanks... Feel free to test off the mainline branch or I can cut an RC sometime in next day or two after fixing this exception.

Reply:
ok fixed here: https://github.com/danmayer/coverband/commit/2ce9355ccc3c8153a72214b19f8c6d7c857881cb